modules/authentication/client-react/access/jwt/components/Loading.jsx

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import { LayoutCenter } from '@module/look-client-react';
 
-export default (
+const Loading = () => (
```

Comment:
Why did you change the `export default` implementation? Is there some need to use this component as a `const`?

Reply:
And why the `App is loading...` component is located in the `auth` module?

Reply:
done